activemq-broker/src/main/java/org/apache/activemq/store/PersistenceAdapterStatistics.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.activemq.store;
+
+import org.apache.activemq.management.TimeStatisticImpl;
+
+public class PersistenceAdapterStatistics {
```

Comment:
This looks fine except I think the rest of the statistics classes extend StatsImpl so you may want to follow that pattern to be consistent.  (For example look at DestinationStatistics)

Reply:
Done! :)